packages/stats/gbstats/bayesian/bandits.py

```diff
@@ -114,25 +110,24 @@ def period_counts(self) -> np.ndarray:
             bandit_period.total_users for bandit_period in self.historical_periods
         ]
         cumulative_counts = cumulative_counts_historical + [self.current_sample_size]
-        period_counts = [cumulative_counts[0]]
-        if self.num_periods > 1:
-            for period in range(1, self.num_periods):
+        period_counts = [cumulative_counts[1]]
+        if self.num_periods_historical > 1:
+            for period in range(1, self.num_periods_historical):
                 period_counts.append(
-                    cumulative_counts[period] - cumulative_counts[period - 1]
+                    cumulative_counts[period + 1] - cumulative_counts[period]
                 )
         return np.array(period_counts)
 
     @property
     def counts_expected(self) -> np.ndarray:
-        counts_expected_by_period = np.empty((self.num_periods, self.num_variations))
+        # historical weights array is right, period counts are wrong, there is extra 0
```

Comment:
What does this comment mean? Does it need to stay here? I'd remove it or maybe make it more clear.